docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,872 @@
+  `system.filesystem`.
+* Models CPU time spent in iowait (`partition_field_name` is `host.name`).
+* Detects unusual increases in cpu time spent in iowait.
+
+Influencers:
+
+* `host.name`
+
+Bucket span: 10m.
+
+Function: `high_mean`.
+
+
+max_disk_utilization_ecs::
+
+* For {metricbeat} data where `event.dataset` is `system.cpu` and 
+  `system.filesystem`.
+* Models disc utilization (`partition_field_name` is `host.name`).
+* Detects unusual increases in disk utilization.
+
+Influencers:
+
+* `host.name`
+
+Bucket span: 10m.
+
+Function: `max`.
+
+
+metricbeat_outages_ecs::
+
+* For {metricbeat} data where `event.dataset` is `system.cpu` and 
+  `system.filesystem`.
+* Models counts of {metricbeat} documents 
+  (`partition_field_name` is `event.dataset`).
+* Detects unusual decreases in {metricbeat} documents.
+
+Influencers:
+
+* `event.dataset`
+
+Bucket span: 10m.
+
+Function: `low_count`.
+
+
+[float]
+[[ootb-ml-jobs-nginx]]
+=== Nginx
+
+These {anomaly-job} wizards appear in {kib} if you use {filebeat} to ship access 
+logs from your http://nginx.org/[Nginx] HTTP servers to {es} and store it using 
+fields and datatypes from the Elastic Common Schema (ECS).
+
+low_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the event rate of http requests. 
+* Detects unusually low counts of HTTP requests compared to the previous event 
+rate.
+
+This job can be created if web access log data exists matching the defined 
+filter.
+
+Bucket span: 15m.
+
+Function: `low_count`.
+
+
+source_ip_request_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high request rates in the HTTP access log 
+  compared to the previous rate. 
+
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_count`.
+
+
+source_ip_url_count_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the event rate of HTTP requests by source IP.
+* Detects source IPs with unusually high distinct count of URLs in the HTTP 
+  access log.
+
+Influencers:
+
+* `source.address`
+
+Bucket span: 1h.
+
+Function: `high_distinct_count`.
+
+
+status_code_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models the occurrences of HTTP response status codes (`partition_field_name` 
+  is `http.response.status_code`).
+* Detects unusual status code rates in the HTTP access log compared to previous 
+  rates.
+
+Influencers:
+
+* `http.response.status_code` 
+* `source.address`
+
+Bucket span: 15m.
+
+Function: `count`.
+
+
+visitor_rate_ecs::
+
+* For HTTP web access logs where `event.dataset` is `nginx.access`.
+* Models visitor rates.
+* Detects unusual visitor rates in the HTTP access log ompared to previous 
+  rates.
+
+Bucket span: 15m.
+
+Function: `non_zero_count`.
+
+
+[float]
+[[ootb-ml-jobs-siem]]
+=== SIEM
+
```

Comment:
I think we should clarify how/where you get these by using text similar to what's in https://www.elastic.co/guide/en/siem/guide/7.6/machine-learning.html#included-jobs and https://www.elastic.co/guide/en/siem/guide/7.6/prebuilt-ml-jobs.html. For example:
```suggestion
These {anomaly-jobs} appear by default in the Anomaly Detection interface of the {siem-guide}/machine-learning.html[SIEM app] in {kib}. They help you automatically detect file system and network anomalies on your hosts.
```
It might be worthwhile to move/single-source the "The jobs are available when... you ship data using Beats, and Kibana is configured with the required index patterns..." etc here too.